hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -279,7 +192,8 @@ generateCommitUTxOs parties = do
             ]
         ]
     commitScript = fromPlutusScript Commit.validatorScript
-    commitDatum = mkCommitDatum party Head.validatorHash utxo
+    commitDatum (Just (_input, _)) = mkCommitDatum party Head.validatorHash utxo (toPlutusCurrencySymbol testPolicyId)
+    commitDatum Nothing = error "Missing utxo"
```

Comment:
Could: follow the style guide and use lambda case (and a new line between where clauses)